test.js

```diff
@@ -125,6 +133,13 @@ test('should write plain text file', async () => {
   assert.equal(readFile('./foo.php'), `/* comments${EOL}version: v1.0.1 */ <? echo <p>hello world</p>; ?>${EOL}`);
 });
 
+test('should write YAML file', async () => {
+  const options = { [namespace]: { out: { file: './foo.yaml' } } };
+  const plugin = factory(Plugin, { namespace, options });
+  await runTasks(plugin);
+  assert.equal(readFile('./foo.yaml'), `version: 1.0.1${EOL}${EOL}`);
```

Comment:
Looking at this result, perhaps the extra `\n` should be removed? I guess `yaml.dump` already adds this.